content/courses/native-onchain-development/paging-ordering-filtering-data-frontend.md

```diff
@@ -103,6 +103,9 @@ Once you’ve fetched accounts with the given data slice, you can use the `sort`
 method to sort the array before mapping it to an array of public keys.
 
 ```tsx
+// 4 bytes are used to store the current length of a string
+const STRING_LENGTH_SPACE = 4;
+
 const accounts = await connection.getProgramAccounts(programId, {
   dataSlice: { offset: 13, length: 15 },
```

Comment:
**These comments are issues with the old code, but we do need to fix them in this update.** See https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#code comments re 'magic numbers'

Why 13, why 15?